detox/src/logger/utils/BunyanLogger.js

```diff
@@ -93,7 +94,7 @@ class BunyanLogger {
           : bunyanStream.stream;
 
         /* istanbul ignore next */
-        return stream.fd !== undefined && !stream.closed;
+        return stream.fd > 2 && !stream.closed;
```

Comment:
This way I check that the given `bunyan` stream is actually a FILE stream, not TTY stream (where fd can be 0, 1, 2).